graphics/src/MeshManager.cc

```diff
@@ -822,6 +822,160 @@ void MeshManager::CreateCamera(const std::string &_name, float _scale)
   mesh->RecalculateNormals();
 }
 
+void MeshManager::CreateCapsule(const std::string &_name,
+                                const float _radius,
+                                const float _length,
+                                const unsigned int _rings,
+                                const unsigned int _segments)
+{
+  if (this->HasMesh(_name))
+  {
+    return;
+  }
+
+  Mesh *mesh = new Mesh();
```

Comment:
Not for here, but I'm going to open an issue about moving away from raw pointers in MeshManagerPrivate.